lib/info.js

```diff
@@ -48,6 +48,18 @@ exports.getBasicInfo = async (id, options) => {
     }
   }
 
+  // Check if there is any captcha form
+  const captchaForm = util.between(body, '<form action="/das_captcha" method="POST">', '</form>');
+  if(captchaForm) {
+    let details = {};
+    details.sitekey = util.between(captchaForm, 'data-sitekey="', '"');
+    details.next = util.between(captchaForm, 'name="next" value="', '"');
+    details.session_token = util.between(captchaForm, 'name="session_token" value="', '"');
+    details.action_recaptcha_verify2 = util.between(captchaForm, 'name="action_recaptcha_verify2" value="', '"');
+    details.largeText = util.between(body, '<p class=\'largeText\'">', '</p>');
+    throw Error("Captcha form received with info: " + JSON.stringify(details));
```

Comment:
is it better to print the json string of `details` here, or to attach all of these properties to the error object?